keyboards/monsgeek/m2/keymaps/via/keymap.c

```diff
@@ -0,0 +1,114 @@
+		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,   KC_B,   KC_N,   KC_M,    KC_COMM, KC_DOT,   KC_SLSH,                          KC_RSFT,                KC_UP,           KC_P1,     KC_P2,    KC_P3,   KC_PENT,
+		KC_LCTL, KC_LALT, KC_LGUI,                  KC_SPC,                           KC_RGUI,   MO(MAC_FN),KC_LEFT,                                      KC_DOWN,  KC_RGHT,            KC_P0,    KC_PDOT),
+
+    [MAC_WASD] = LAYOUT( /* Base */
+		KC_ESC,  KC_BRID, KC_BRIU, KC_MCTL, KC_LPAD, KC_F5,  KC_F6, KC_MPRV, KC_MPLY, KC_MNXT, KC_MUTE, KC_VOLD,  KC_VOLU,      KC_DEL,                                   KC_HOME,  KC_END,  KC_PGUP,   KC_PGDN,
+		KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,    KC_8,    KC_9,     KC_0,     KC_MINS,    KC_EQL,     KC_BSPC,                                 KC_NUM,  KC_PSLS,  KC_PAST,   KC_PMNS,
+		KC_TAB,  KC_Q,    KC_UP,   KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,    KC_I,    KC_O,     KC_P,     KC_LBRC,    KC_RBRC,    KC_BSLS,                                 KC_P7,     KC_P8,    KC_P9,   KC_PPLS,
+		KC_CAPS, KC_LEFT, KC_DOWN, KC_RIGHT,KC_F,   KC_G,   KC_H,   KC_J,    KC_K,    KC_L,     KC_SCLN,  KC_QUOT,               KC_ENT,                                  KC_P4,     KC_P5,    KC_P6,
+		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,   KC_B,   KC_N,   KC_M,    KC_COMM, KC_DOT,   KC_SLSH,                          KC_RSFT,                KC_UP,           KC_P1,     KC_P2,    KC_P3,   KC_PENT,
+		KC_LCTL, KC_LALT, KC_LGUI,                  KC_SPC,                           KC_RGUI,   MO(MAC_FN),KC_LEFT,                                      KC_DOWN,  KC_RGHT,            KC_P0,    KC_PDOT),
+
+    [MAC_FN] = LAYOUT( /* FN */
+		KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,  KC_F5,  KC_F6, KC_F7,   KC_F8,  KC_F9,     KC_F10,  KC_F11,     KC_F12,     KC_DEL,                                   KC_HOME,  KC_END,  KC_PGUP,   KC_PGDN,
+		KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,  KC_7,    KC_8,   KC_9,      KC_0,    RGB_SPD,    RGB_SPI,    KC_BSPC,                                  KC_NUM,  KC_PSLS,  KC_PAST,   KC_PMNS,
+		KC_TAB,  KC_Q,TG(MAC_WASD),KC_E,    KC_R,   KC_T,   KC_Y,  KC_U,    KC_INS, DF(WIN_B), KC_PSCR, KC_LBRC,    KC_RBRC,    RGB_MOD,                                  KC_P7,     KC_P8,    KC_P9,   KC_PPLS,
+		KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,  KC_J,    KC_K,   RGB_TOG,   KC_SCLN, KC_QUOT,                RGB_HUI,                                  KC_P4,     KC_P5,    KC_P6,
+		KC_LSFT, KC_Z,    KC_X,    KC_CALC, KC_V,   KC_B,   KC_N,  KC_MUTE, KC_VOLD,KC_VOLU,  KC_SLSH,                          KC_RSFT,               RGB_VAI,           KC_P1,     KC_P2,    KC_P3,   KC_PENT,
+		KC_LCTL, KC_LALT, KC_LGUI,                  KC_SPC,                           KC_RGUI,  MO(MAC_FN),RGB_SAD,                                   RGB_VAD,  RGB_SAI,            KC_P0,    KC_PDOT)
+
+};
+// clang-format on
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```

Comment:
`QK_CLEAR_EEPROM` (aka `EE_CLR`) clears the eeprom, and also triggers the via refresh